core/rfb.js

```diff
@@ -413,6 +417,24 @@ export default class RFB extends EventTargetMixin {
         }
     }
 
+    get requestLocalWakelock() {
```

Comment:
This seems like an entirely frontend thing. Any reason to put stuff in the `RFB` object?